stack.yaml

```diff
@@ -5,6 +5,10 @@ packages:
 
 extra-deps:
 - rio-0.1.21.0@rev:0
+# TODO: temporary
```

Comment:
Please update this, as well as the other stack config files

Reply:
Done